### PR TITLE
Plot points

### PR DESCRIPTION
**Pull request by vishvanath45**
Things completed - 

- Takes output file as argument and plot points.
- User can also give file containing all coordinates along with output file to plot them along with convex hull.
- User can give `-s` flag to save the generated plot directly.

### CODE REVIEW

**Comment by Compro-Prasad** [comment]
It should be a closed polygon.

**Comment by Compro-Prasad** [comment]
Mention a `requirements.txt` or `Pipfile` for easy installation of dependencies.

**Comment by vishvanath45** [comment]
> Mention a requirements.txt or Pipfile for easy installation of dependencies.  

This can go as another PR.

File: plot.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+				sys.exit(1)
+		f.close()
+		plt.scatter(inp_x, inp_y)
+
+if(sys.argv[no_of_args-1] == '-s'):
+	filename = datetime.datetime.now()
+	filename = "plot-"+str(filename)+".jpg"
+	plt.savefig(filename)
+else:
+	plt.show()
```
Comment (Compro-Prasad) [inline plot.py]: Add newline at end of file

File: plot.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	with open(sys.argv[2], "r") as f:
+		lines_2_to_end = f.readlines()[1:]
+		for line in lines_2_to_end:
+			try:
+				a,b = map(float, line.split())
+				inp_x.append(a)
+				inp_y.append(b)
+			except:
+				sys.stderr.write("ERROR! - File is malformed. It should have two space separated \
+					real numbers per line., with first line containing no. of values.")
```
Comment (Compro-Prasad) [inline plot.py]: `no. of points`.

**Comment by Compro-Prasad** [comment]
Great work. Merged it. Good luck on your next PR.

**Pull request merged**